src/integrationTest/sam.test.ts

```diff
@@ -28,6 +28,7 @@ import { insertTextIntoFile } from '../shared/utilities/textUtilities'
 import { sleep } from '../shared/utilities/timeoutUtils'
 import globals from '../shared/extensionGlobals'
 import { closeAllEditors } from '../test/testUtil'
+import { randomUUID } from 'crypto'
```

Comment:
nit: other places where a uuid is needed use this package: https://github.com/aws/aws-toolkit-vscode-staging/blob/c617c3180f0dd8c6313f8f121e46176fd7752a83/src/shared/telemetry/util.ts#L11

Maybe we don't actually need that package? Else it's good to be consistent